rootfs/etc/cont-init.d/01-config.sh

```diff
@@ -185,8 +194,8 @@ if [[ "$(yq --output-format=json e '(.. | select(tag == "!!str")) |= envsubst' "
     if [[ "$(_jq '.hidefiles')" != "null" ]] && [[ -n "$(_jq '.hidefiles')" ]]; then
       echo "hide files = $(_jq '.hidefiles')" >> /etc/samba/smb.conf
     fi
-    if [[ "$(_jq '.recycle')" != "null" ]] && [[ -n "$(_jq '.recycle')" ]]; then
-      echo "vfs objects = recycle" >> /etc/samba/smb.conf
+    if [[ -n "$(_jq '.recycle')" ]] && [[ "$(_jq '.recycle')" == "yes" ]]; then
```

Comment:
I think you need to check nullable first:

```suggestion
    if [[ "$(_jq '.recycle')" != "null" ]] && [[ "$(_jq '.recycle')" = "yes" ]]; then
```

similar to https://github.com/crazy-max/docker-samba/pull/116/files#diff-4ba17048e76227dbde3fbc98302d14f92284e27360ac7ff1f5230740c79cf2fdR190